### ISSUE TITLE
sell of imported PDF not editable

### ISSUE THREAD

**Issue by christen90** [buchen/portfolio#2005]
**Describe the bug**
In VL-FondsDepos from ebase there is no reference account. Taxes and fees are applied via a SALE of shares. The resulting amount is 0. I'm not able to save it as a SALE action. However the PDF importer manages to create such bookings.

**To Reproduce**
1. Import Data from _ebase_auszug06.txt_ test case #2004 
2. Buchung editieren (Verkauf)
3. as long as Gutschrift = 0 - item is not saveable

**Expected behavior**
imported items are editable

**Screenshots**
![image](https://user-images.githubusercontent.com/26897965/105610587-a5aae600-5db0-11eb-9b65-59b51574223a.png)

**Desktop (please complete the following information):**
 - Windows
 - Version 20H2

**Solution Ideas**
- GUI: accept 0 as amount if TAX or FEE is filled
- change importer to create OUTBOUND_DELIVERY instead of SALE

**Comment by zr40** [comment]
> GUI: accept 0 as amount if TAX or FEE is filled

Or rather, always accept 0. This would help with removing expired option positions.

**Comment by chirlu** [comment]
That’s an importer bug, I’d say.

@zr40: You can (and should) use “outbound delivery”.

**Comment by zr40** [comment]
Outbound delivery suffers from the same problem. It doesn't accept 0.

**Comment by chirlu** [comment]
> Outbound delivery suffers from the same problem. It doesn't accept 0.

You haven’t actually tried it, have you?

**Comment by zr40** [comment]
I actually have tried. It says "Input required for 'Sub-Total'" and the Save button is disabled.

**Comment by chirlu** [comment]
Oh, that’s interesting, because it is meant to work and it does work for me.

After some experiments, it seems that the (blocking) error only appears if you enter 0 into the subtotal field. If, instead, you enter 0 into either of “Quote” or “Value of outbound delivery”, the subtotal will be calculated as 0, and the error is raised. That’s a bug.

**Comment by christen90** [comment]
With outbound delivery it can be done:
![image](https://user-images.githubusercontent.com/26897965/105736673-593ce300-5f35-11eb-8156-f1bda7c5ecab.png)

Which booking type is more applicable for _Vermögenswirksame Leistungen_? This specific booking does not contain a reference account. I fear, that an outbound booking is more suited and the importer has to be changed... 
Also the purchase of _VL-FondsDepot_ could fit better into OUTBOUND_DELIVERY. Maybe its better to make a difference between normal depot and _VL-FondsDepot_ and split the importer into 2

Anyway it should not be possible, that the importer is able to create bookings which can not be replicated by using the gui.

**Comment by zr40** [comment]
> Oh, that’s interesting, because it is meant to work and it does work for me.
> 
> After some experiments, it seems that the (blocking) error only appears if you enter 0 into the subtotal field directly. If, instead, you enter 0 into either of “Quote” or “Value of outbound delivery”, the subtotal will be calculated as 0, and the error isn’t raised. That’s a bug.

Indeed, it works for outbound delivery. Thanks! For inbound delivery however I can't get it to accept 0.

**Comment by chirlu** [comment]
> For inbound delivery however I can't get it to accept 0.

Yes, that’s correct. It doesn’t make sense as it would, e.g., mean infinite performance.

**Comment by zr40** [comment]
Not for my use case, which is written options that have expired.
